src/app/(app)/page.js

```diff
@@ -1,4 +1,5 @@
-import { TherapiesSection, FAQSection } from '@components/MainPageSections';
+import { FAQSection, TherapiesSection } from '@components/MainPageSections';
+import { DistrictSearch } from '@components/DistrictSearch';
```

Comment:
🟠 maybe place this component in 'MainPageSections' like 2 previous?

Reply:
Moved it to the MainPageSections folder